### ISSUE TITLE
google analysis always post warnings , why ?

### ISSUE THREAD

**Issue by marksnoopy** [igrigorik/ga-beacon#30]
![image](https://cloud.githubusercontent.com/assets/2253657/10502884/283986c2-7323-11e5-9adb-bb34504b3092.png)

**Comment by olivdee** [comment]
same here... i checked everything and cannot find an error in my email newsletters code... suppose something else went wrong / is down here at ga-beacon? here some more info:

**Tracking Code Mismatch**
The default URL https://ga-beacon.appspot.com/, for property UA-########-# has no tracking code for this property, although it does have tracking codes for other properties:

UA-3769691-2

If your Tracking ID and your Default URL setting are out of sync, you don't get valid data in your Analytics account for the Default URL.

Either update the tracking code to use the Tracking ID for property ON Newsletter Tracking, or open the Property Settings to update the Default URL for property ON Newsletter Tracking to be https://ga-beacon.appspot.com/.

**Comment by igrigorik** [comment]
You probably want to set your default URL to `ga-beacon.appspot.com/UA-XXXXX-XX/`. I believe that should fix it..

**Comment by marksnoopy** [comment]
i deleted the property and create a new one , it work well. thank you .

**Comment by igrigorik** [comment]
Cool, glad to hear!

**Comment by olivdee** [comment]
i changed the property as you suggested an hour ago, but i dont get any hits...
should i delete it and create a new one as marksnoopy?
and then also change the default URL as well?
@marksnoopy : did you do both?

**Comment by marksnoopy** [comment]
i delete the account and property ,create a new account and peoperty , then
i get a new property .

replace the old property in my gist page , save the gist and refresh the
gist page , you will see the visit data in the real-time page of google
analysis .

**Comment by olivdee** [comment]
@marksnoopy : so did you completely deleted your GA account? and did you set the default URL as ilya suggested above?
@igrigorik : is this really necessary???

**Comment by marksnoopy** [comment]
not GA account , just web site account . 

![image](https://cloud.githubusercontent.com/assets/2253657/10595731/3c3c9ec2-770e-11e5-89de-c2c31dd3ba0c.png)

**Comment by olivdee** [comment]
thanks mark for clarifiyng. will try first only to change the default URL as ilya suggested. if thats not successful i will delete the account

**Comment by olivdee** [comment]
somehow the email / newsletter tracking does not work as expected...
we did run several tests with 5 different email addresses, for example:
mail1@companyname.com: opened on Thunderbird PC1
mail2@gmail.com               : opened on Android and later on as well on PC1 gmail browser interface
mail3@companyname.com : opened on Thunderbird PC2
mail4@companyname.com : opened on Thunderbird PC1
mail5@someotherdomain.com : opened on  PC1 someotherdomain browser interface

so my count would suggest 5 unique page views and 6 page views (because of the double gmail opening), but GA only shows 2 unique page views (which are definitely from prior opening any of the above mails, thus i guess false positives from the newsletter-tool?) and 3 page views.

am i missing something out? or is there a bug (implementation or how ga-beacon works)?
is this because we are opening those mails from within the same office / IP?

**Comment by olivdee** [comment]
this is all very weird... after all that testing we thought that the unique page views are generated by the newsletter tool, so we finally did send the newsletter with 2 hours prior to midnight, in order to be able to filter in GA from the next day on, and count simple page views (usually a newsletter is opened only once). so we thought this might work with us. but after sending the NL (to more than 2500 subscribers), we got only 9 unique page views (as opposed to the expected > 2500) and up to today not a single normal page view! even though we were on the subscriber list ourselves and opened the NL on our PCs... 

any idea what could be failing here?
if i copy the ga-beacon URL from the NLs sourcecode and insert it to a browser, the counting works and i get one UNIQUE PAGE VIEW and one NORMAL PAGE VIEW after lets say 15 min...
